README.md

```diff
@@ -1,2 +1,34 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1) Minimum ceph version should be Ceph-CSI supported
+2) Existing volumes should be dynamically provisioned.
+3) Stop the application pods that are consuming the flex volume(s) that need to be converted.
+4) `ceph-common`packages, for rbd ceph commands.
+
+## Steps to Mirgrate
+1) Create Ceph-CSI storageClass in which you want to migrate:
+   1) Example: `kubectl create -f https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml`
+2) 1) Create rbac required to run migrator pod
+   1) `kubectl create -f tests/manifests/rbac.yaml`
+3) Create migrator pod
+   1) `kubectl create -f tests/manifests/toolbox.yaml`
+4) Run command to start migration:
+   ### To migrate **single** PVC:
+      1) `<binary_name> --pvc=<pvc-name> --pvc-namespace=<pvc-namespace> --destinationstorageclass=<storageclass-name-to-migrate-in> <--rook-namespace=rook-ceph> <--ceph-cluster-namespace=ceph-cluster-namespace>`
```

Comment:
- What's the actual binary name?
- Let's use square brackets for optional arguments and angle brackets for required arguments
```suggestion
      1) `pvc-migrator --pvc=<pvc-name> --pvc-namespace=<pvc-namespace> --destinationstorageclass=<storageclass-name-to-migrate-in> [--rook-namespace=rook-ceph] [--ceph-cluster-namespace=ceph-cluster-namespace]`
```

Reply:
`pvc-migrator` is the name we have planned for. Currently, it `main` what we get after `go build main.go`